webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric-criteria-readonly.js

```diff
@@ -24,51 +24,64 @@ export class SakaiRubricCriteriaReadonly extends RubricsElement {
       <div class="criterion style-scope sakai-rubric-criterion-readonly">
         <div id="sort-criterion">
           ${this.criteria.map(c => html`
-            <div id="criterion_row_${c.id}" class="criterion-row">
-              <div tabindex="0" class="criterion-detail">
-                <h4 class="criterion-title">${c.title}</h4>
-                <p>${unsafeHTML(c.description)}</p>
-                ${this.weighted ? html`
-                    <div class="criterion-weight">
-                        <span>
-                          <sr-lang key="weight">Weight</sr-lang>
-                        </span>
-                        <span>${c.weight.toLocaleString(this.locale)}</span>
-                        <span>
-                          <sr-lang key="percent_sign">%</sr-lang>
-                        </span>
-                    </div>`
-                  : ""
-                }
+            ${this.isCriterionGroup(c) ? html`
+              <div id="criterion_row_${c.id}" class="criterion-row criterion-group">
+                <div class="criterion-detail">
+                  <h4 class="criterion-title">${c.title}</h4>
+                  <p>${unsafeHTML(c.description)}</p>
+                </div>
               </div>
-              <div class="criterion-ratings">
-                <div class="cr-table">
-                  <div class="cr-table-row">
-                  ${c.ratings.map(r => html`
-                    <div tabindex="0" title="${tr("rating_title")}: ${r.title}. ${tr("rating_description")}: ${r.description}. ${tr("point_value")}: ${r.points.toLocaleString(this.locale)}" class="rating-item" id="rating_item_${r.id}">
-                      <h5 class="criterion-item-title">${r.title}</h5>
-                      <p>${r.description}</p>
-                      <span class="points">
-                        ${this.weighted && r.points > 0 ? html`
-                            <b>
-                              (${parseFloat((r.points * (c.weight / 100)).toFixed(2)).toLocaleString(this.locale)})
-                            </b>`
-                          : ""
-                        }
-                        ${r.points.toLocaleString(this.locale)}
-                        <sr-lang key="points">Points</sr-lang>
-                      </span>
+            ` : html`
+              <div id="criterion_row_${c.id}" class="criterion-row">
+                <div tabindex="0" class="criterion-detail">
```

Comment:
why is this div tabbable?

Reply:
@adrianfish could you review, why this is focusable please? I'm not sure whats the purpose here